AddKeptnResourceTask/AddKeptnResourceTaskV1/task.json

```
@@ -14,7 +14,7 @@
   "version": {
     "Major": 1,
     "Minor": 1,
-    "Patch": 0
+    "Patch": 6

```

we can probably jump just 1 patch version forward ;)
```suggestion
    "Patch": 1
```